application-analytics-default/src/main/java/com/xwiki/analytics/internal/RowEvolutionJsonNormaliser.java

```diff
@@ -60,11 +60,12 @@ public String getIdentifier()
     }
 
     /**
-     * Transforming the json object returned by Matomo into an array of jsons to make it easier to use in javascript.
+     * Transforming the json object returned by Matomo into an array of jsons to make it easier to use in javascript and
+     * adding a date to each entry to be able to idenitfy for what period the statistics are.
```

Comment:
`Transform the Matomo JSON response into an array of JSONs to simplify usage. Add a date to each entry to be able to identify from what period the statistics are.`